articles/aks/openfaas.md

```diff
@@ -25,43 +25,46 @@ In order to complete the steps within this article, you need the following.
 * Azure CLI installed on your development system.
 * Git command-line tools installed on your system.
 
-## Get OpenFaaS
+## Add the OpenFaaS helm chart repo
 
-Clone the OpenFaaS project repository to your development system.
+OpenFaaS maintains its own helm charts to keep up to date with all the latest changes.
 
 ```azurecli-interactive
-git clone https://github.com/openfaas/faas-netes
-```
-
-Change into the directory of the cloned repository.
-
-```azurecli-interactive
-cd faas-netes
+helm repo add openfaas https://openfaas.github.io/faas-netes/
+helm repo update
 ```
 
 ## Deploy OpenFaaS
 
 As a good practice, OpenFaaS and OpenFaaS functions should be stored in their own Kubernetes namespace.
 
-Create a namespace for the OpenFaaS system.
+Create a namespace for the OpenFaaS system and functions:
 
 ```azurecli-interactive
-kubectl create namespace openfaas
+kubectl apply -f https://raw.githubusercontent.com/openfaas/faas-netes/master/namespaces.yml
 ```
 
-Create a second namespace for OpenFaaS functions.
+Generate a password for the OpenFaaS UI Portal and REST API:
 
 ```azurecli-interactive
-kubectl create namespace openfaas-fn
+# generate a random password
+PASSWORD=$(head -c 12 /dev/urandom | shasum| cut -d' ' -f1)
+
+kubectl -n openfaas create secret generic basic-auth \
+--from-literal=basic-auth-user=admin \
+--from-literal=basic-auth-password="$PASSWORD"
 ```
 
+This password will prevent authorized access to your OpenFaaS gateway which will be exposed to the public Internet via a cloud LoadBalancer.
```

Comment:
This password will prevent unauthorized access ?

Reply:
Yes

Reply:
I can reword it.